include/circt/Dialect/ESI/ESIManifest.td

```diff
@@ -169,6 +169,23 @@ def AppIDHierNodeOp : ESI_Op<"manifest.hier_node", [
   }];
 }
 
+def SymbolConstantsOp : ESI_Op<"manifest.consts", [
```

Comment:
I've been bashed for excessive succinctness before, so i'm here going to push back on calling this `consts` instead of `constants` :).